kinde_fastapi/framework/fastapi_framework.py

```
@@ -133,30 +137,80 @@ async def login(request: Request):
+
+            user_id = request.session.get('user_id') or str(uuid.uuid4())
+
             try:
-                # Generate a unique user ID for the session
-                logger.warning(f"[Callback] Request session is: {request.session}")
-                user_id = request.session.get("user_id") or str(uuid.uuid4())
-                
-                # Use OAuth's handle_redirect method to process the callback
-                logger.warning(f"[Callback] code[{code}] state[{state}] user_id[{user_id}]")
+                assert self._oauth is not None
                 result = await self._oauth.handle_redirect(code, user_id, state)
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)